lib/ansible/playbook/role/requirement.py

```diff
@@ -189,6 +189,8 @@ def scm_archive_role(src, scm='git', name=None, version='HEAD', keep_scm_meta=Fa
 
         def run_scm_cmd(cmd, tempdir):
             try:
+                stdout = None
+                stderr = None
```

Comment:
this will create a concatenation error if they are still 'None' in the exception handling, initialize as '' (empty string)